course-mw/course-actors-common/src/main/java/org/sunbird/learner/util/CourseBatchUtil.java

```diff
@@ -34,6 +41,17 @@
   private static ElasticSearchService esUtil = EsClientFactory.getInstance(JsonKey.REST);
   private static ObjectMapper mapper = new ObjectMapper();
   private static LoggerUtil logger = new LoggerUtil(CourseBatchUtil.class);
+  private static final SimpleDateFormat DATE_FORMAT = ProjectUtil.getDateFormatter();
+  private static final SimpleDateFormat DATE_FORMAT_SIMPLE = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
SimpleDateFormat - Concurrency will create issues for the format. Please use it as local variable.